MovieAppPetProject/MovieAppPetProject/GetMovieDetailsByID/GetMovieDetails-ViewModel.swift

```diff
@@ -61,6 +59,7 @@ class MovieDetailsViewModel {
             switch result {
             case .success(let details):
                 self?.movieDetails = details
+                self?.view?.updateUI()
             case .failure(let error):
                 print(error)
             }
```

Comment:
proper error management and removal of print statements to come in dedicated error management PR